src/ocean/text/arguments/TimeIntervalArgs.d

```diff
@@ -517,7 +517,7 @@ unittest
 
 ******************************************************************************/
 
-public bool isTimeInterval ( cstring value )
+private bool isRelativeTime ( cstring value )
```

Comment:
This is a change to the public API that would need to be in a major release.

Reply:
I don't think that there is anyone who uses this function...

Reply:
This is an open source library so we can't actually say that for sure.

Reply:
Correct. A change like this must always be considered a breaking change. If the function must be renamed, consider using a deprecation.

Reply:
done